libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/common/ThriftEnumUtils.java

```diff
@@ -48,10 +48,11 @@ private ThriftEnumUtils() {
     );
     //! Enumeration "fancy name" mappings
     private static final ImmutableMap<ComponentType, String> MAP_COMPONENT_TYPE_STRING = ImmutableMap.of(
-            ComponentType.OSS, "Open Source Software" ,
-            ComponentType.COTS, "Commercial Off The Shelf" ,
-            ComponentType.INTERNAL, "Internal" ,
-            ComponentType.FREESOFTWARE, "Free Software");
```

Comment:
I don't have a preference. I can see how Internal and Inner Source could be more familiar to Siemens employees.
@maximilianhuber , does Bosch have a name for this kind of components (Inner Source)?